### PR TITLE
Feature/put to cache by lang code

### PR DESCRIPTION
**Pull request by an42rus**

### CODE REVIEW

File: rosetta/angular_translation_storage.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    from django.core.cache import get_cache
+    cache = get_cache(rosetta_settings.ROSETTA_CACHE_NAME)
+
+try:
+    import importlib
+except ImportError:
+    from django.utils import importlib
+
+
+class BaseAngularTranslationStorage(object):
```
Comment (korvyashka) [inline rosetta/angular_translation_storage.py:18]: If you use it basically as interface - you can use abc.ABCMeta

File: rosetta/management/commands/compilemessages.py [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+        for basedir in basedirs:
+            locale_dirs = filter(os.path.isdir, glob.glob('%s/*' % basedir))
+            all_locales.extend(map(os.path.basename, locale_dirs))
+
+        # Account for excluded locales
+        locales = locale or all_locales
+        locales = set(locales) - set(exclude)
+
+        for basedir in basedirs:
+            if not locales:
```
Comment (korvyashka) [inline rosetta/management/commands/compilemessages.py]: What is point of this check inside cycle? it seems is not populated inside cycle.

File: rosetta/utils.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    """Convert po object to dictionary data structure (ready for JSON).
+    """
+    result = {}
+
+    for entry in po:
+        if entry.obsolete:
+            continue
+
+        if entry.msgstr:
+            result[entry.msgid] = entry.msgstr
```
Comment (korvyashka) [inline rosetta/utils.py]: it looks like it can be replaced with one line dict generator.

File: rosetta/views/get_translation.py [hunk 4]
```diff
@@ -0,0 +1,34 @@
+import json
+from django.conf import settings
+from rest_framework.exceptions import NotFound
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from rest_framework.permissions import AllowAny
```
Comment (korvyashka) [inline rosetta/views/get_translation.py]: Do you think rest_framework should be part of the rosetta?

**Review by korvyashka — CHANGES_REQUESTED** [review]

File: rosetta/management/commands/compilemessages.py (hunk 2 above)
Comment (an42rus) [reply rosetta/management/commands/compilemessages.py]: oh... of cource it should be outside...

File: rosetta/angular_translation_storage.py (hunk 1 above)
Comment (an42rus) [reply rosetta/angular_translation_storage.py:18]: yes. i know, but i think it's not critical now

File: rosetta/views/get_translation.py (hunk 4 above)
Comment (an42rus) [reply rosetta/views/get_translation.py]: hm... it seems that django rest not needed here. removed

**Pull request merged**